### PR TITLE
Add a handler for empty string logs

### PR DESCRIPTION
**Pull request by Mubangizi**
#### What this PR does
- Returns `No logs found` message for apps with empty strings

### CODE REVIEW

**Review by MutegekiHenry — APPROVED** [review]

**Review by neelxie — APPROVED** [review]
Thanks @Mubangizi

File: app/controllers/app.py [hunk 1]
```diff
@@ -1631,7 +1631,7 @@ def post(self, project_id, app_id):
                 pod_infor = f'type\tstatus\treason\t\t\tmessage\n----\t------\t------\t\t\t------\nwaiting\tfailed\t{reason}\t{message}'
                 pods_logs.append(pod_infor)
 
-        if pods_logs == []:
+        if pods_logs == [] or len(pods_logs[0]) == 0:
```
Comment (inno-asiimwe) [inline app/controllers/app.py:1634]:
```suggestion
        if not pods_logs or not pods_logs[0]:
```

**Review by inno-asiimwe — CHANGES_REQUESTED** [review]
@Mubangizi we can make the change more pythonic

**Review by inno-asiimwe — APPROVED** [review]
LGTM

**Pull request merged**